### ISSUE TITLE
wondering about your ubiased proof 

### ISSUE THREAD

**Issue by jack-pan-ai** [usaito/unbiased-implicit-rec-real#7]
Hello, in your proof, **Proposition 3.1. (Bias of WMF estimator) The bias of the estimator used in WMF is represented as follows**
![Screenshot from 2021-05-17 15-45-36](https://user-images.githubusercontent.com/35600925/118450929-4ab64980-b727-11eb-9b78-fc6abba861c1.png)

$\delta$ is as well concerned with \gamma (relevance probability). 

The weighted matrix factorization loss function **cannot be calculated directly** without estimating \gamma (relevance probability). So, may I know how do you use this loss function in the real-world setup?

![Screenshot from 2021-05-17 15-49-07](https://user-images.githubusercontent.com/35600925/118451983-59e9c700-b728-11eb-8582-f24b2f856cb4.png)

ps: sorry, I do not know how to type math formula in the reply

**Comment by usaito** [comment]
@PancheLone Thanks for your interest in our work! Please make sure that Eq. (6) does not depend on \gamma. It just uses implicit feedback Y (, which is observable) . Instead of using the unobservable quantity \gamma. How to estimate \gamma in the loss functions using only observable data is the key in our paper.

**Comment by jack-pan-ai** [comment]
Thanks for your prompt reply a lot! But according to your definition in Eq.(4), we have to know P(R_{ui}=1), which is \gamma, so that we can calculate the \delta in your Eq.(6).

![Screenshot from 2021-05-18 10-06-49](https://user-images.githubusercontent.com/35600925/118579514-d03a0800-b7c0-11eb-9e0c-d8bb9add1897.png)
![Screenshot from 2021-05-18 10-10-33](https://user-images.githubusercontent.com/35600925/118579784-522a3100-b7c1-11eb-96a8-b0c499f9bea3.png)


Thanks for your help in advance, and looking forward to your reply!

**Comment by usaito** [comment]
@PancheLone In Eq.(6) (and the loss functions of some other methods), \delta^{(1)} and \delta^{(0)} are used, both of which do not depend on \gamma, right? For example, \delta^{(1)} = - \log (\hat{R}) where \hat{R} is a predicted value, not the true \gamma. 

\Eq. (4) is the quantity that we want to maximize, and is not used in the training process.

**Comment by jack-pan-ai** [comment]
Ooooo, it's the predicted value!!!, It seems like I got your idea; predicted value means to use the learned embedding vector!!! Thank you very much for your kind explanation. 

Moreover, I also followed your twitter, and hope for more valuable and interesting work from you in the future!

**Comment by usaito** [comment]
Great!
